src/content/en/tools/chrome-devtools/javascript/breakpoints.md

```diff
@@ -0,0 +1,238 @@
+    </td>
+  </tr>
+  <tr>
+    <td><a href="#event-listeners">Event listener</a></td>
+    <td>
+      Want to investigate the code that runs after an event, such as
+      <code>click</code>, is fired.
+    </td>
+  </tr>
+  <tr>
+    <td><a href="#exceptions">Exception</a></td>
+    <td>
+      Want to pause on the line of code that is throwing a caught or
+      uncaught exception.
+    </td>
+  </tr>
+</table>
+
+## Line-of-code breakpoints {: #loc }
+
+Use a line-of-code breakpoint when you know the exact region of code that you
+need to investigate. DevTools *always* pauses before this line of code is
+executed.
+
+To set a line-of-code breakpoint in DevTools:
+
+1. Click the **Sources** tab.
+1. Open the file containing the line of code you want to break on.
+1. Go the line of code.
+1. To the left of the line of code is the line number column. Click on it. A
+   blue icon appears on top of the line number column.
+
+<figure>
+  <img src="imgs/loc-breakpoint.png"
+       alt="A line-of-code breakpoint."
+  <figcaption>
+    <b>Figure 1</b>: The blue icon on top of <b>29</b> indicates that there's
+    a line-of-code breakpoint set on that line.
+  </figcaption>
+</figure>
+
+### Conditional line-of-code breakpoints {: #conditional-loc }
+
+Use a conditional line-of-code breakpoint when you know the exact region of
+code that you need to investigate, but you want to pause only when some
+other condition is true.
+
+To set a conditional line-of-code breakpoint:
+
+1. Click the **Sources** tab.
+1. Open the file containing the line of code you want to break on.
+1. Go the line of code.
+1. To the left of the line of code is the line number column. Right-click on
+   it.
+1. Select **Add conditional breakpoint**. A dialog displays underneath the
+   line of code.
+1. Enter your condition in the dialog.
+1. Press <kbd>Enter</kbd> to activate the breakpoint. An orange icon appears
+   on top of the line number column.
+
+<figure>
+  <img src="imgs/conditional-loc-breakpoint.png"
+       alt="A conditional line-of-code breakpoint."
+  <figcaption>
+    <b>Figure 2</b>: The orange icon on top of <b>32</b> indicates that there's
+    a conditional line-of-code breakpoint set on that line.
+  </figcaption>
+</figure>
+
+### Manage line-of-code breakpoints {: #manage-loc }
+
+Use the **Breakpoints** pane to disable or remove line-of-code breakpoints from
+a single location.
+
+* Check the checkbox next to an entry to disable that breakpoint.
+* Right-click over an entry to remove that breakpoint.
+* Right-click anywhere in the **Breakpoints** pane to deactivate all
+  breakpoints, disable all breakpoints, or remove all breakpoints. Disabling
+  all breakpoints is equivalent to unchecking each one. Deactivating all
+  breakpoints instructs DevTools to ignore all line-of-code breakpoints, but to
+  also maintain preserve their enabled state so that they are in the same
+  state as before when you re-activate them.
```

Comment:
I had to go old-school to confirm. I.e., I had to look in my actual physical copy of Webster's. 'Reactivate' (no hyphen) is the proper form.